aten/src/ATen/quantized/Quantizer.h

```diff
@@ -43,7 +43,8 @@ using QuantizerPtr = c10::intrusive_ptr<Quantizer>;
  */
 struct CAFFE2_API Quantizer : public c10::intrusive_ptr_target {
   const QScheme qscheme_;
-  explicit Quantizer(QScheme qscheme) : qscheme_(qscheme) {}
+  const ScalarType scalar_type_;
```

Comment:
why do you need to keep scalar type in quantizer if it's already present in the tensor?